src/main/webapp/WEB-INF/views/UserPage.html

```diff
@@ -0,0 +1,388 @@
+                    <a class="btn btn-edit pull-right" href="EditUserPage.html">
+                        <h4><i class="glyphicon glyphicon-pencil"></i>Edit</h4>
+                    </a>
+                </div>
+                <div class="panel-body">
+                    <div class="row">
+                        <div class="col-lg-4 col-md-4 col-sm-4  hidden-xs">
+                            <img class="img-responsive img-rounded big-photo"
+                                 src="http://vev.ru/uploads/images/00/07/64/2011/10/12/c26b1b.jpg"/>
+                        </div>
+                        <div class="col-lg-8 col-md-8 col-sm-8 col-xs-12">
+                            <div class="panel-info">
+                                <div class="panel-body">
+                                </div>
+                                <table class="table">
+                                    <tr>
+                                        <th>First name:</th>
+                                        <td>User`s First Name</td>
+                                    </tr>
+                                    <tr>
+                                        <th>Last name:</th>
+                                        <td>User`s Last Name</td>
+                                    </tr>
+                                    <tr>
+                                        <th>Email:</th>
+                                        <td>User`s email</td>
+                                    </tr>
+                                    <tr>
+                                        <th>Phone:</th>
+                                        <td>User`s phone</td>
+                                    </tr>
+                                    <tr>
+                                        <th>Address:</th>
+                                        <td>User`s address</td>
+                                    </tr>
+                                    <tr>
+                                        <th></th>
+                                        <td></td>
+                                    </tr>
+                                </table>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+<!--------------------------------------------------------------------------------------->
+<div class="container">
+    <div class="row">
+        <div class="col-xs-12">
+            <a href="#favourites" class="btn btn-info btn-block" data-toggle="collapse">Favourites</a>
+            <div class="collapse" id="favourites">
+                <div class="well">
+                    <div class="panel-info">
+                        <div class="panel-body">
+                            <div class="row masonry" data-columns>
+                                <div class="card">
+                                    <a href="#"><img class="card-img-top img-responsive"
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Add an "alt" attribute to this image. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)